app/build.gradle.kts

```diff
@@ -90,10 +90,13 @@ dependencies {
     implementation(libs.kotlinx.coroutines.android)
     implementation(libs.androidx.datastore.preferences)
     implementation(libs.jgrapht.core)
+    implementation(libs.jgrapht.io)
+    implementation(libs.moshi.kotlin)
     implementation(libs.moshi)
     implementation(libs.moshi.adapters)
     implementation(libs.moshi.kotlin)
-
+    implementation(libs.graph.support)
```

Comment:
didn't graph support turn out to be nonviable for our purposes? same question for libs toml file